grafana-plugin/src/containers/IntegrationContainers/ExpandedIntegrationRouteDisplay/ExpandedIntegrationRouteDisplay.tsx

```diff
@@ -63,6 +70,22 @@ interface ExpandedIntegrationRouteDisplayState {
   routeIdForDeletion: string;
 }
 
+enum ROUTING_OPTION {
```

Comment:
We have similar places in codebase but it would be good to use PascalCase for enum name and its keys just as [official docs](https://www.typescriptlang.org/docs/handbook/enums.html) is doing. Maybe we could bring some simple eslint rule for that

Reply:
We didn't have any pattern for this really, or at least I don't remember we agreed on naming. Changed all enums to pascal case